README.md

```diff
@@ -240,9 +240,21 @@ player.core.activeContainer.on(Clappr.Events.CONTAINER_STATE_BUFFERING, function
 See all existing events on Clappr [here](https://github.com/clappr/clappr-core/blob/master/src/base/events.js#L227).
 
 #### plugins
-An array used to pass external plugins instances to Clappr. You can pass plugins of any category in this array.
+An object used to config external plugins instances and plugins behaviors to Clappr.
 
-Example:
+```javascript
+{
+  plugins: {
+    core: [CorePlugin],
+    container: [ContainerPlugin],
+    playback: [Playbacks],
+    disableExternalPluginsLoadPrecedence: false,
```

Comment:
I think it's confuse to assume a negation in a flag variable, I'd say that, if possible, something like: `externalPlaybacksLoadPrecedence` when it's `false` it won't follow the exterenal playback load precedence... and then it'd be good if we document the defaults.

Reply:
Sure! I was praying that someone would come up with a better suggestion for the names of these options. Done on this commit 31e9f7f3dcc7980ee6889faff3bb33ce6b0e208d